ouroboros-consensus-cardano/test/cardano-test/Main.hs

```diff
@@ -16,7 +17,7 @@ import           Test.Util.TestEnv (defaultMainWithTestEnv,
                      defaultTestEnvConfig)
 
 main :: IO ()
-main = do
+main = withStdTerminalHandles $ do
   hSetBuffering stdout LineBuffering
   hSetEncoding stdout utf8
```

Comment:
I think you can now remove the
```haskell
hSetEncoding stdout utf8
```
line

Reply:
Done in bff359cbb6b53ee9ff774f9ac040fa455c86998d.